WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderHtmlUtils.java

```diff
@@ -13,6 +13,15 @@
         void onTagFound(String tag, String src);
     }
 
+    // regex for matching oriwidth attributes in tags
+    private static final Pattern ORIGINAL_WIDTH_ATTR_PATTERN = Pattern.compile(
```

Comment:
`ORIGINAL_WIDTH_ATTR_PATTERN` and `ORIGINAL_HEIGHT_ATTR_PATTERN` are the same. Perhaps just a single `ORIGINAL_SIZE_ATTR_PATTERN`?